.github/workflows/check_tutorials.yml

```diff
@@ -1,5 +1,5 @@
 # Run unit, integration, and functional tests.
-name: Pytest
+name: Check tutorials
```

Comment:
What happened here? The name does not seem to match the comment above.

Reply:
Will fix comment.